MAUI/Barcode-Generator/one-dimensional-symbology.md

```diff
@@ -0,0 +1,450 @@
+
+## Code39 Extended
+
+* `Code39 Extended` symbology is an extended version of Code39 that supports all 128 ASCII characters set. So, it encodes lower case alphabets (a-z) as well as special characters.
+
+{% tabs %}
+
+{% highlight xaml hl_lines="5" %}
+
+    <barcode:SfBarcodeGenerator Value="051091"
+                               ShowText="True"
+                               HeightRequest="150">
+        <barcode:SfBarcodeGenerator.Symbology>
+            <barcode:Code39Extended Module="2" />
+        </barcode:SfBarcodeGenerator.Symbology>
+    </barcode:SfBarcodeGenerator>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="4" %}
+
+    SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+    barcode.HeightRequest = 150;
+    barcode.Value = "051091";
+    barcode.Symbology = new Code39Extended() { Module = 2 };
+    barcode.ShowText = true;
+    this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator Code39 Extended Symbology](images/one-dimensional-symbology/maui-code39-extended-symbology.png)
+
+As like `Code39`, the `Code39 Extended` supports the `EnableCheckSum` property.
+
+## Code93
+
+`Code93` is designed to complement and enhance `Code39`. It represents the complete ASCII character set by using a combination of 2 characters. It is a continuous, variable-length symbology and produces a denser code.
+
+* Encodes character set of uppercase alphabets (A-Z), digits (0-9), and special characters such as asterisk (*), dash (-), dollar ($), percent (%), Space, dot (.), slash (/), plus (+) and so on.
+* The asterisk (*) is not a true encoding character, but it is the start and stop symbol for `Code93` Symbology.
+
+{% tabs %}
+
+{% highlight xaml hl_lines="5" %}
+
+    <barcode:SfBarcodeGenerator Value="01234567"
+                                ShowText="True"
+                                HeightRequest="150">
+        <barcode:SfBarcodeGenerator.Symbology>  
+            <barcode:Code93 Module="2" />
+        </barcode:SfBarcodeGenerator.Symbology>
+    </barcode:SfBarcodeGenerator>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="4" %}
+
+    SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+    barcode.HeightRequest = 150;
+    barcode.Value = "01234567";
+    barcode.Symbology = new Code93() { Module = 2 };
+    barcode.ShowText = true;
+    this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator Code93 Symbology](images/one-dimensional-symbology/maui-code93-symbology.png)
+
+## Code128
+
+`Code 128` is a highly efficient, high-density linear barcode symbology that allows the encoding of alphanumeric data. It is capable of encoding full ASCII character set and extended character sets. This symbology contains the checksum digit for verification and the barcode can also be verified character-by-character for the parity of each data byte.
```

Comment:
It is capable of encoding full ASCII character sets and extended character sets.

Reply:
changes updated.